cmd/res/configuration.toml

```diff
@@ -127,10 +132,10 @@ Labels = []
 UseMessageBus = true
   [Device.Discovery]
   Enabled = true
-  Interval = "1h"
+  Interval = "30s"
 
 [Driver]
-RtspServerHostName = "localhost"
+RtspServerHostName = "192.168.10.4"
```

Comment:
undo your local dev changes. Note, for development it is often easier to use env overrides so that you do not change the file